packages/react-draggable-dialog/package.json

```diff
@@ -2,11 +2,16 @@
   "name": "@fluentui-contrib/react-draggable-dialog",
   "version": "0.0.1",
   "private": true,
+  "dependencies": {
+    "@dnd-kit/core": "^6.1.0",
```

Comment:
They are all the latest. `@dnd-kit/core` is the main library and the other two are "extensions" that are compatible with the main, so it can be different.